presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/SuiteRun.java

```diff
@@ -216,11 +219,19 @@ public TestRunResult executeSuiteTestRun(String suiteName, SuiteTestRun suiteTes
             testRunOptions.environment = suiteTestRun.getEnvironmentName();
             testRunOptions.testArguments = suiteTestRun.getTemptoRunArguments(environmentConfig);
             testRunOptions.testJar = suiteRunOptions.testJar;
-            String targetDirectory = format("%s-%s-%s", suiteName, environmentConfig.getConfigName(), suiteTestRun.getEnvironmentName());
-            testRunOptions.reportsDir = Paths.get("presto-product-tests/target/reports/" + targetDirectory);
-            testRunOptions.logsDirBase = logsDirBase.resolve(targetDirectory);
+            String suiteRunId = suiteRunId(suiteName, suiteTestRun, environmentConfig);
+            testRunOptions.reportsDir = Paths.get("presto-product-tests/target/reports/" + suiteRunId);
+            testRunOptions.logsDirBase = logsDirBase.resolve(suiteRunId);
             return testRunOptions;
         }
+
+        private static String suiteRunId(String suiteName, SuiteTestRun suiteTestRun, EnvironmentConfig environmentConfig)
+        {
+            List<String> testArguments = suiteTestRun.getTemptoRunArguments(environmentConfig);
+            // suite-2 contains two singlenode environment test runs - use tempto arguments to uniquely identify each test run
+            String testArgumentsHash = Hashing.sha256().hashString(Joiner.on("-").skipNulls().join(testArguments), Charsets.UTF_8).toString();
```

Comment:
this is not useful.
Can we make it a single digit number. Only present if there actually is a conflict. 
Or alternatively you can just put an index of `SuiteTestRun` in the list in the string. Right?